README.md

```diff
@@ -45,11 +45,13 @@ echo "error page" >> client/dist/error.html
 **Third**, run the plugin, and visit your new website!
 
 ```
-serverless client deploy [--stage $STAGE] [--region $REGION]
```

Comment:
Did we lose the stage functionality? Honestly I hadn't used it before but I noticed you cut it out?

Reply:
@linusmarco Lol, you're totally right. Yeah we can add this back in later if we want. But taking it out for now makes the most sense if it didn't do anything.